Gemfile

```diff
@@ -57,7 +57,7 @@ gem 'carrierwave-aws'
 gem 'clam_scan'
 gem 'combine_pdf'
 gem 'config'
-gem 'connect_vbms', git: 'https://github.com/adhocteam/connect_vbms', tag: 'v0.2.0.rc', require: 'vbms'
+gem 'connect_vbms', git: 'https://github.com/department-of-veterans-affairs/connect_vbms.git', branch: 'master', require: 'vbms'
```

Comment:
@stevenjcumming Did you mean to put this change in with the debian bookworm test?

Reply:
Yes, the connect_vbms repo needed to be upgraded to ruby 3.2+. But I can't push to VA connect_vbms so I needed to fork it.